rpc/client/local/local.go

```diff
@@ -127,6 +127,10 @@ func (c *Local) ConsensusParams(ctx context.Context, height *int64) (*ctypes.Res
 	return core.ConsensusParams(c.ctx, height)
 }
 
+func (c *Local) Events(ctx context.Context, req *ctypes.RequestEvents) (*ctypes.ResultEvents, error) {
+	return core.Events(c.ctx, req.Filter.Query, req.MaxItems, req.Before, req.After, req.WaitTime)
```

Comment:
Out of curiosity: #7982 implements the new functionality directly in this file, whereas this PR moves it inside the `core` package. Was there a reason?

Reply:
To allow reuse in here and [routes](https://github.com/tendermint/tendermint/blob/0dce59e60d936e8e35b09d7ab43cc7032a417c09/rpc/core/routes.go#L13), while https://github.com/tendermint/tendermint/pull/7982 with parseParams refactor only use in [local](https://github.com/tendermint/tendermint/blob/5deb9689ff6deebf43e13cae618785c18d751c1c/rpc/client/local/local.go#L133).